lib/pxenforcer.js

```diff
@@ -383,6 +383,9 @@ class PxEnforcer {
             pass_reason: ctx.passReason,
             ...ctx.additionalFields,
         };
+        if (ctx.serverInfoRegion) {
```

Comment:
If you add it to both `page_requested` and `block`, why not include it in `getActivityDetails()` instead?